docs/workflows/kserve-predictor.md

```diff
@@ -13,12 +13,125 @@ The KServe predictor step expects a model URL to be supplied as input, pointing
 The predictor performs the following tasks:
 
 - downloads the model locally from the MLflow artifact store
-- if so instructed, it auto-detects the model format based on the information stored in the MLflow artifact store and decides which KServe predictor engine to use for it. Otherwise, it validates the model format against the type of predictor engine specified as input.
+- if so instructed (i.e. the `predictor` input parameter is omitted or explicitly set to `auto`), it auto-detects the model format based on the information stored in the MLflow artifact store and decides which KServe predictor engine to use for it. Otherwise, it validates the model format against the type of predictor engine specified as input.
```

Comment:
I think you should state explicitly that the values for predictor here are `tensorflow`, `sklearn` and `triton` and that `triton` is selected when `onnx` or `keras` are found in the MLmodel...

Reply:
Ah, OK, now I see you have it later in the text

Reply:
This is just an overview, what you're asking for is already covered in the next section